front-end/src/redux/basketSlice.js

```diff
@@ -0,0 +1,44 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+export const basketSlice = createSlice({
+    name: 'basket',
+    initialState: {
+        products: []
+    },
+    reducers: {
+        setInitialBasket: (state, action) => {
+            state.products = [...state.products, ...action.payload]
+            console.warn(
```

Comment:
Is this needed?

Reply:
If I don't spread the initial items they are placed as one element, and I thought it's easier to read this way.

Reply:
I was talking about the console.warn